bin/regenerate_sqlx_data.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
```

Comment:
Maybe this should go into `src/rust/bin` instead? Keep it closer to the source?

Reply:
very reasonable